compiler/src/iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -62,10 +62,10 @@ static FailureOr<Operation *> getRootOp(func::FuncOp funcOp) {
 /// scf::tileUsingSCFForOp expects the num of tile sizes = num of loops. This
 /// method returns a proper tile sizes vector for each op during tiling.
 static SmallVector<Value> buildTileSizesForOp(OpBuilder &b, Operation *op,
-                                              SmallVector<int64_t> tileSizes) {
+                                              ArrayRef<int64_t> tileSizes) {
   auto tilingOp = cast<TilingInterface>(op);
 
-  SmallVector<int64_t> newTileSizes = tileSizes;
+  SmallVector<int64_t> newTileSizes = llvm::to_vector(tileSizes);
```

Comment:
[optional] we can use the constructor, so this can also be

```suggestion
  SmallVector<int64_t> newTileSizes(tileSizes);
```